package.json

```
@@ -31,7 +31,7 @@
   "homepage": "https://github.com/UKHomeOffice/asl#readme",
   "dependencies": {
     "@asl/pages": "^31.8.1",
```

```suggestion
    "@asl/pages": "^31.8.2",
```
+ re-run npm install and push new lock file as well.